File: pgm/ABC_JavaScript/ABC_155_A.js
function main(input) {
    var arr = input.replace(/\r?\n/g, "").split(" ");
    const A = arr[0], B = arr[1], C = arr[2];
    var flg = false;
    if (A == B && B != C) flg = true;
    if (B == C && C != A) flg = true;
    if (C == A && A != B) flg = true;
    if (flg) console.log("Yes");
    else console.log("No");
}
main(require('fs').readFileSync('/dev/stdin', 'utf8'));

File: pgm/ABC_R/ABC_155_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
Yes <- "Yes"
No <- "No"
one <- 1L
two <- 2L
blank <- ""
space <- " "
ABC <- strsplit(x = arr[one], split = space)[[one]] |>
    as.integer()
table <- table(ABC)
if (table |>
        length() == two) {
    Yes |>
        cat("\n", sep = blank)
} else {
    No |>
        cat("\n", sep = blank)
}
q("no")